src/module.c

```diff
@@ -10,21 +10,14 @@
 #include "redismodule.h"
 #include "version.h"
 #include "commands/commands.h"
-#include "graph/graph_type.h"
-#include "index/index_type.h"
-#include "stores/store_type.h"
+#include "graph/serializers/graphcontext_type.h"
 #include "util/thpool/thpool.h"
 #include "arithmetic/agg_funcs.h"
+#include "arithmetic/arithmetic_expression.h"
```

Comment:
Why do we need to include arithmetic_expression.h here?

Reply:
For `AR_RegisterFuncs()`